### PR TITLE
SDK-1915: SDK docs updates

### PR DESCRIPTION
**Pull request by sergeyspryker**
## PR Description
https://spryker.atlassian.net/browse/SDK-1915

Updated SDK docs

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/sdk/dev/spryker-sdk.md [hunk 1]
```diff
@@ -7,36 +7,41 @@ redirect_from:
...
+## Getting started
 
-```bash
-spryker-sdk list
-```
+To get an overview on the available capabilities of the Spryker SDK please run **spryker-sdk list**.
 
-You can execute any task by running `spryker-sdk <task-id>` from the project root folder. To get information about the options that you can pass into a task, run `bin/consolespryker-sdk <task-id> -h`.
+Any task can be executed by running **spryker-sdk \<task-id\>** from project root folder.
+Using **bin/consolespryker-sdk \<task-id\> -h** will give a description on what options can be passed into the task.
```
Comment (DmytroKlymanSpryker) [inline docs/sdk/dev/spryker-sdk.md:47]: Space is missing in `bin/consolespryker-sdk`

**Review by DmytroKlymanSpryker — APPROVED** [review]

File: docs/sdk/dev/spryker-sdk.md [hunk 2]
```diff
@@ -7,36 +7,41 @@ redirect_from:
 ---
 The Spryker SDK aims to provide a single entry point to accelerate your productivity while working with Spryker. The Spryker SDK provides tools to validate existing code, implement new features with Spryker, and go live with your project.
 
+## Requirements
+- Ensure auth file is available for composer (https://getcomposer.org/doc/articles/authentication-for-private-packages.md).
```
Comment (AlexSlawinski) [inline docs/sdk/dev/spryker-sdk.md:11]:
```suggestion
- Make sure an auth file is available for the [Composer](https://getcomposer.org/doc/articles/authentication-for-private-packages.md).
```

File: docs/sdk/dev/spryker-sdk.md [hunk 3]
```diff
@@ -7,36 +7,41 @@ redirect_from:
...
+- **Coreutils** should be installed (only for the mac users).
```
Comment (AlexSlawinski) [inline docs/sdk/dev/spryker-sdk.md:12]:
```suggestion
- If you're using a MacOS computer, install **Coreutils**.
```

File: docs/sdk/dev/spryker-sdk.md [hunk 4]
```diff
@@ -7,36 +7,41 @@ redirect_from:
...
+- Follow the installer's instructions.
+- Alias **spryker-sdk** should be set and **SPRYKER_SDK_PATH** env variable should be exported.
 
-## Getting started
 
-To set up your local settings, run this command:
-```bash
-spryker-sdk sdk:setting:set
+### Installation command
+Install sdk into the current dir
```
Comment (AlexSlawinski) [inline docs/sdk/dev/spryker-sdk.md:31]:
```suggestion
Install sdk into the current dir.
```

**Review by AlexSlawinski — APPROVED** [review]

**Pull request merged**